stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -171,17 +173,21 @@ public class MessageListView : ConstraintLayout, IMessageListView {
         }
     private val DEFAULT_REACTION_VIEW_CLICK_LISTENER =
         ReactionViewClickListener { message: Message ->
-            MessageMoreActionDialog(
-                context,
-                channel,
-                message,
-                currentUser,
-                style,
-                onMessageEditHandler,
-                onMessageDeleteHandler,
-                onStartThreadHandler,
-                onMessageFlagHandler
-            ).show()
+            context.getFragmentManager()?.let {
+                // TODO: pass a real MessageItem instead of mock
+                val mockMessageItem = MessageItem(
+                    message.apply {
+                        latestReactions.forEach { it.type = ReactionType.LOVE.type }
+                        ownReactions.forEach { it.type = ReactionType.LOVE.type }
+                    },
```

Comment:
Would be good to update the rendered test bo be sure we don't release it to production.
We could use `copy()` method instead, to not update the one that comes from the listener.
something like:
```suggestion
                    message.copy(
                        latestReactions = message.latestReactions.forEach { it.type = ReactionType.LOVE.type },
                        ownReactions, = message.ownReactions.forEach { it.type = ReactionType.LOVE.type },
                        text = "MOCKED TEXT: ${message.text}"
                    },
```

Reply:
It is just is shallow copy, right?

Reply:
Yep, creating a copy to not update the previous message object